CHANGELOG.rst

```diff
@@ -6,6 +6,130 @@
 Synapse Changelog
 *****************
 
+v2.156.0 - 2023-12-08
+=====================
+
+Model Changes
+-------------
+- Updates to the ``infotech``, ``ou``,  and ``risk`` models.
+  (`#3436 <https://github.com/vertexproject/synapse/pull/3436>`_)
+  (`#3438 <https://github.com/vertexproject/synapse/pull/3438>`_)
+  (`#3446 <https://github.com/vertexproject/synapse/pull/3447>`_)
+  (`#3447 <https://github.com/vertexproject/synapse/pull/3447>`_)
+
+  **New Properties**
+
+  ``it:av:scan:result``
+    The form had the following properties added to it:
+
+      ``target:ipv4``
+        The IPv4 that was scanned to produce the result.
```

Comment:
```suggestion
        The IPv4 address that was scanned to produce the result.
```

Reply:
This is the property doc. Should the property doc be updated?

Reply:
per discussion we'll update the doc